tests.gs

```diff
@@ -100,6 +101,29 @@ function testSemVer () {
   });
 }
 
+/**
+ * Test whether dateWithTimezone() handles DST as expected.
+ */
+function testDSTCorrectness () {
+  var date, now;
+
+  now = new Date();
+
+  // Winter: +0200 expected.
+  date = dateWithTimezone(now.getFullYear, 10, 1, 0, 0, 0, 'Europe/Athens');
```

Comment:
For this not to fail it needs `s/now.getFullYear/now.getFullYear()/`

Reply:
My fault: I had caught this and modified my local script, but forgot to amend the commit.